src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -30,6 +31,26 @@ private static bool IsNet70OrGreater()
 
         private static string IncludeExpectedOutput(string expectedOutput) => IsNet70OrGreater() ? expectedOutput : null;
 
+        private static MetadataReference MscorlibRefWithoutSharingCachedSymbols
```

Comment:
>private static MetadataReference MscorlibRefWithoutSharingCachedSymbols [](http://example.com/codeflow?start=8&length=71)

Can this be a `static readonly` field with initializer so we don't need to copy the value to a local in methods that create more than one compilation?

Reply:
There's only two tests that use this. What are you trying to achieve?

Reply:
I was mistaken. I thought this was used in more tests, and that all tests could share a common reference.